src/com/duckduckgo/mobile/android/activity/OnboardingActivity.java

```diff
@@ -115,12 +109,7 @@ public void onPageScrollStateChanged(int state) {
         });
         viewPager.setAdapter(adapter);
 
-        int[] backgroundColors = new int[] {
-                ContextCompat.getColor(this, PrivacyFragment.BACKGROUND_COLOR),
-                ContextCompat.getColor(this, NoAdsFragment.BACKGROUND_COLOR),
-                ContextCompat.getColor(this, NoTrackingFragment.BACKGROUND_COLOR),
-                ContextCompat.getColor(this, RightFragment.BACKGROUND_COLOR),
-                ContextCompat.getColor(this, EndOnboardingFragment.BACKGROUND_COLOR)};
+        int[] backgroundColors = Onboarding.getBackgroundColors(this);
```

Comment:
I think the problem here is that the Adapter for the OnboardingActivity ViewPager is still a FragmentPagerAdapter, maybe it's worth convert that one too?